src/System.Private.Xml/tests/XmlResolver/System.Xml.XmlResolver.Tests/XmlPreloadedResolverConstructionTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Xml.Resolvers;
+using Xunit;
+
+namespace System.Xml.XmlResolver.Tests
+{
+    public class XmlPreloadedResolverConstructionTests
+    {
+        [Fact]
+        public void XmlResolverWithDefaultConstructor()
+        {
+            var resolver = new XmlPreloadedResolver();
+            Assert.Equal(14, resolver.PreloadedUris.Count());
+        }
+
+        [Fact]
+        public void XmlResolverWithKnownDtdsConstructor()
+        {
+            var resolver = new XmlPreloadedResolver(XmlKnownDtds.Xhtml10);
+            Assert.Equal(12, resolver.PreloadedUris.Count());
+            Assert.True(resolver.PreloadedUris.Any(u => u.OriginalString.Equals("-//W3C//DTD XHTML 1.0 Strict//EN")));
```

Comment:
I'd put the expected values in the array and just for(each) loop on them

Reply:
also count should be checked with the array's length so that we are sure there is no diff